### PR TITLE
Ca 93 user can answer questions

### PR DESCRIPTION
**Pull request by DawsonDease**

### CODE REVIEW

File: navigation/AppNavigator.js [hunk 1]
```diff
@@ -20,7 +21,7 @@ const AuthNavigator = () => {
   });
 
   return (
-    <Stack.Navigator initialRouteName="Account">
+    <Stack.Navigator initialRouteName="Question">
```
Comment (katiemcgoff) [inline navigation/AppNavigator.js]: Should our initial route be Question? I know we're in progress on the nav and hooking everything up together but I think this may be problematic for @JaycobDuffel

File: screens/AllVideosScreen.js [hunk 2]
```diff
@@ -44,10 +43,8 @@ export default function AllVideosScreen() {
           auth: apiConfig.auth,
         })
         .then((data) => {
-          console.log(data.data.results);
           setData(data.data.results);
           setLoading(false);
-          console.log(loading);
         })
```
Comment (katiemcgoff) [inline screens/AllVideosScreen.js:48]: I like these console logs being removed, but in general we should not check in changes that are unrelated to the ticket in question. It's fine here but let's avoid it moving forward.

File: __tests__/Question_tests.js [hunk 3]
```diff
@@ -0,0 +1,62 @@
+import "react-native";
+import React from "react";
+import Question from "../components/Question";
+import Button from "react-native-paper";
```
Comment (JaycobDuffel) [inline __tests__/Question_tests.js]: Unused import

**Review by JaycobDuffel — CHANGES_REQUESTED** [review]

**Review by katiemcgoff — APPROVED** [review]

**Review by JaycobDuffel — APPROVED** [review]
